restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/telephony/DialActionOrganizationSBCTest.java

```diff
@@ -0,0 +1,1079 @@
+        assertLastOperationSuccess(mariaCall);
+        assertTrue(mariaCall.waitForAuthorisation(3000));
+
+        assertTrue(shoaibCall.waitForIncomingCall(5000));
+        assertTrue(shoaibCall.sendIncomingCallResponse(100, "Trying-Shoaib", 1800));
+        assertTrue(shoaibCall.sendIncomingCallResponse(180, "Ringing-Shoaib", 1800));
+        String receivedBody = new String(shoaibCall.getLastReceivedRequest().getRawContent());
+        assertTrue(shoaibCall.sendIncomingCallResponse(Response.OK, "OK-Shoaib", 3600, receivedBody, "application", "sdp", null,
+                null));
+
+        assertTrue(mariaCall.waitOutgoingCallResponse(5 * 1000));
+        int responseMaria = mariaCall.getLastReceivedResponse().getStatusCode();
+        assertTrue(responseMaria == Response.TRYING || responseMaria == Response.RINGING);
+
+        Dialog mariaDialog = null;
+
+        if (responseMaria == Response.TRYING) {
+            assertTrue(mariaCall.waitOutgoingCallResponse(5 * 4000));
+            assertEquals(Response.RINGING, mariaCall.getLastReceivedResponse().getStatusCode());
+            mariaDialog = mariaCall.getDialog();
+        }
+
+        assertTrue(mariaCall.waitOutgoingCallResponse(5 * 1000));
+        assertEquals(Response.OK, mariaCall.getLastReceivedResponse().getStatusCode());
+        assertTrue(mariaCall.getDialog().equals(mariaDialog));
+        mariaCall.sendInviteOkAck();
+        assertTrue(mariaCall.getDialog().equals(mariaDialog));
+
+        assertTrue(!(mariaCall.getLastReceivedResponse().getStatusCode() >= 400));
+
+        assertTrue(shoaibCall.waitForAck(3000));
+
+        //Talk time ~ 3sec
+        Thread.sleep(3000);
+        shoaibCall.listenForDisconnect();
+        assertTrue(mariaCall.disconnect());
+
+        assertTrue(shoaibCall.waitForDisconnect(5 * 1000));
+        assertTrue(shoaibCall.respondToDisconnect());
+
+        Thread.sleep(5000);
+
+        //Check live calls
+        int liveCalls = MonitoringServiceTool.getInstance().getStatistics(deploymentUrl.toString(), superAdminAccountSid, adminAuthToken);
+        int liveCallsArraySize = MonitoringServiceTool.getInstance().getLiveCallsArraySize(deploymentUrl.toString(), superAdminAccountSid, adminAuthToken);
+        logger.info("&&&&& LiveCalls: "+liveCalls);
+        logger.info("&&&&& LiveCallsArraySize: "+liveCallsArraySize);
+        assertEquals(0, liveCalls);
+        assertEquals(0,liveCallsArraySize);
+        
+
+    }
+
+    /**
+     * testClientsCallEachOtherDifferentOrganization
+     * given clients:
+     *
+     * maria belongs to org: org2.restcomm.com
+     * alice belong to org: org3.restcomm.com
+     *
+     * test case: maria calls alice.
+     *
+     * result: call do not go through
+     *
+     * @throws ParseException
+     * @throws InterruptedException
+     */
+    @Test
+    public void testClientsCallEachOtherDifferentOrganization() throws ParseException, InterruptedException {
```

Comment:
this should be in "exception" category IMO

Reply:
I'm not aware of what the exception category is. I copied the DialActionOrganizationTest.java, not sure if it was present initially there either.